experimental/packages/api-logs/package.json

```diff
@@ -26,7 +26,6 @@
     "lint:fix": "eslint . --ext .ts --fix",
     "version": "node ../../../scripts/version-update.js",
     "watch": "tsc --build --watch tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
-    "precompile": "lerna run version --scope $(npm pkg get name) --include-dependencies",
```

Comment:
Hmm, what's the reason for removing the precompile script there - did it cause troubles? :thinking:

Reply:
This was by accident. For some reason lerna did not get installed when I bootstrapped the whole repo, and I did not have time to troubleshoot. I am putting it back.